### PR TITLE
Add Vault Alerts and Update Ticket Template

### PR DESCRIPTION
**Pull request by Shaad7**
- Add Vault Alerts
- Add Vault Ticket Template
- Update Database Ticket Template

Signed-off-by: Shaad7 <abdullah.alshaad@appscode.com>

### CODE REVIEW

File: charts/vaultserver/ticket-template.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+> Please refer to [KubeVault Docs](https://kubevault.com/docs/latest/guides/) for more about KubeVault.
+
+Let's say you have VaultServer instance in namespace demo.
+```bash
+kubectl get vaultserver -n <ns>  // will list all the database in a namesapce     
+$ kubectl get vaultserver -n demo 
+  NAME           VERSION   STATUS   AGE
+  sample-vs      6.2.5     Ready    1h11m
+```
+There are four different db phase you may see in KubeVault managed Database.
```
Comment (TasdidurRahman) [inline charts/vaultserver/ticket-template.md:12]: db phase ?

Comment (TasdidurRahman) [reply charts/vaultserver/ticket-template.md:12]: kubevault managed database ?

Comment (TasdidurRahman) [reply charts/vaultserver/ticket-template.md:12]: fix all the `databases` and `db`s

File: charts/vaultserver/ticket-template.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    kubectl logs -n <ns> <pod-name> -c vault
+    # If the vaultserver is sealed
+    kubectl logs -n <ns> <pod-name> -c vault-unsealer
+    ```
+- Operator logs:
+    ```bash
+    kubectl logs -n <kubevault-ns> <operator-pod-name>
+    ```
+
+**NotReady:** Database Phase NotReady means none of the servers are working properly. There could several possible reasons for that, maybe something is misconfigured,
```
Comment (TasdidurRahman) [inline charts/vaultserver/ticket-template.md:88]: fix `database`

File: charts/vaultserver/ticket-template.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+This template is designed to contact with AppsCode with troubleshooting and support in general. To troubleshoot and find what to do let's get started with the vaultserver phase.
+
+> Please refer to [KubeVault Docs](https://kubevault.com/docs/latest/guides/) for more about KubeVault.
+
+Let's say you have VaultServer instance in namespace demo.
```
Comment (raihankhan) [inline charts/vaultserver/ticket-template.md:5]: Might change it to > Let's say you have VaultServer instance provisioned in a namespace.
Generalize the namespace.

File: charts/vaultserver/ticket-template.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+```bash
+kubectl get vaultserver -n <ns>  // will list all the database in a namesapce
```
Comment (raihankhan) [inline charts/vaultserver/ticket-template.md]: > databases

File: charts/vaultserver/ticket-template.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
+This template is designed to contact with AppsCode with troubleshooting and support in general. To troubleshoot and find what to do let's get started with the vaultserver phase.
+
+> Please refer to [KubeVault Docs](https://kubevault.com/docs/latest/guides/) for more about KubeVault.
+
+Let's say you have VaultServer instance in namespace demo.
+```bash
+kubectl get vaultserver -n <ns>  // will list all the database in a namesapce     
+$ kubectl get vaultserver -n demo
```
Comment (raihankhan) [inline charts/vaultserver/ticket-template.md:8]: `$ kubectl get vaultserver -n <ns>`

File: charts/vaultserver/ticket-template.md (hunk 1 above)
Comment (raihankhan) [inline charts/vaultserver/ticket-template.md]: > There are four different db phases you might see in KubeVault managed Databases.

**Review by raihankhan — CHANGES_REQUESTED** [review]

File: charts/vaultserver/remedy.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    - Revoke the unused Tokens
+    - Check if Any instance creating more tokens than necessary.
+
+### KubeVault Operator Alerts
+
+- #### AppPhaseNotReady
+    - Contact AppsCode team
+- #### AppPhaseCritical
+    - If any `VaultOpsRequest` is ongoing on same database, Wait until it completes.
+    - check for `vault` container's logs on the node to see if anything unusual.
```
Comment (raihankhan) [inline charts/vaultserver/remedy.md:40]: > Check

File: charts/vaultserver/ticket-template.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+maybe the server is killed, Replication errors, or something miscellaneous.
+To resolve this, first we need to know what exactly happened. Checking the logs from operator and pod containers, describing the VaultServer object and pods is a recommended way to start debugging. Restarting the pod might sometime solve the issue. But, before forcing a cluster fail-over and recover,
+there might be a need for human intervention to know what will be the best way to resolve it.
+
+In that case please contact AppsCode with the following information.
+
+- VaultServer object:
+    ```bash
+    kubectl get vs -n <ns> <vaultserver-object-name> -oyaml
+    Kubectl get pod -n <demo> <pod-name> -oyaml
```
Comment (raihankhan) [inline charts/vaultserver/ticket-template.md:98]: > `    Kubectl get pod -n <ns> <pod-name> -oyaml
`

File: charts/vaultserver/ticket-template.md (hunk 5 above)
Comment (Shaad7) [reply charts/vaultserver/ticket-template.md:8]: Followed common pattern

**Review by TasdidurRahman — APPROVED** [review]
LGTM

**Pull request merged**